libraries/tools/kotlin-stdlib-gen/src/templates/Mapping.kt

```diff
@@ -441,4 +441,17 @@ object Mapping : TemplateGroupBase() {
         }
     }
 
+    val f_withEach = fn("withEach(operation: T.() -> Unit)") {
+      include(Iterables)
```

Comment:
Consider providing it for the same receivers as `forEach`.
Also it makes sense to make it inline same as `forEach` (note that different overloads have different inline modifiers)